src/components/ContactList.tsx

```
@@ -0,0 +1,20 @@
+  contacts: Array<string>;
+}
+
+export default function ContactList({ contacts }: ContactListProps) {
+  return (
+    <div className="flex flex-col items-start bg-gray-800 border-1 border-gray-500 rounded-lg p-3">
+      <p>Name</p>
+      <hr className="bg-gray-500 w-full" />
+      {contacts.map((contact) => {
+        return (
+          <>
+            <p className="text-blue-500 py-4">{contact}</p>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)